content/docs/command-reference/plots/show.md

```diff
@@ -139,6 +169,35 @@ file:///Users/usr/src/plots/logs.csv.html
 
 ![](/img/plots_show.svg)
 
+<details>
+
+### Expand to see example using TSV metrics file
```

Comment:
Please move this right after the YAML section and call it **Expand for TSV format**.

Reply:
Done